### PR TITLE
Add changelog entry for v2.7.1

### PR DESCRIPTION
**Pull request by astorije**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
...
+## v2.7.1 - 2018-02-18
+
+For more details, [see the full changelog](https://github.com/thelounge/lounge/compare/v2.7.0...v2.7.1) and [milestone](https://github.com/thelounge/lounge/milestone/29?closed=1).
+
+This releases mainly fixes bugs that were introduced by previous versions, and comes with very minor improvements to the UI.
+This simply ensures we did not leave any unattended bugs before going for The Lounge v3.
+
+### Changed
+
+- Show channels for only active server in autocomplete ([#1993](https://github.com/thelounge/lounge/pull/1993) by [@milindl](https://github.com/milindl))
```
Comment (xPaw) [inline CHANGELOG.md]: Autocomplete channels only for current network

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
...
+  - `lodash` ([#2032](https://github.com/thelounge/lounge/pull/2032))
+  - `commander` ([#2038](https://github.com/thelounge/lounge/pull/2038), [#2041](https://github.com/thelounge/lounge/pull/2041))
+  - `urijs` ([#2053](https://github.com/thelounge/lounge/pull/2053))
+  - `mime-types` ([#2067](https://github.com/thelounge/lounge/pull/2067))
+
+### Fixed
+
+- Prevent user context menu from opening while selecting text ([#1955](https://github.com/thelounge/lounge/pull/1955) by [@xPaw](https://github.com/xPaw))
+- Fix timestamp tooltips not aligning correctly with timestamps ([#1999](https://github.com/thelounge/lounge/pull/1999) by [@astorije](https://github.com/astorije))
+- Set `start_url` in `manifest.json` ([#2010](https://github.com/thelounge/lounge/pull/2010) by [@xPaw](https://github.com/xPaw))
```
Comment (xPaw) [inline CHANGELOG.md]: [...] so that The Lounge always opens correct window.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
...
+- Do not statically serve the index template prior to rendering it ([#1979](https://github.com/thelounge/lounge/pull/1979) by [@astorije](https://github.com/astorije))
+- Reload query windows on server restart ([#2019](https://github.com/thelounge/lounge/pull/2019) by [@McInkay](https://github.com/McInkay))
```
Comment (xPaw) [inline CHANGELOG.md]: Persist query windows between server restarts

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
 
 <!-- New entries go after this line -->
 
+## v2.7.1 - 2018-02-18
+
+For more details, [see the full changelog](https://github.com/thelounge/lounge/compare/v2.7.0...v2.7.1) and [milestone](https://github.com/thelounge/lounge/milestone/29?closed=1).
+
+This releases mainly fixes bugs that were introduced by previous versions, and comes with very minor improvements to the UI.
+This simply ensures we did not leave any unattended bugs before going for The Lounge v3.
```
Comment (xPaw) [inline CHANGELOG.md]: Mention that we fixed unread markers bug which in turn also fixes memory leak if you had lounge open for long periods of time (e.g. overnight)

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
...
+- Fix queries going to lobby if the network name matches user name ([#2037](https://github.com/thelounge/lounge/pull/2037) by [@xPaw](https://github.com/xPaw))
+- Fix default theme not being correct ([#2033](https://github.com/thelounge/lounge/pull/2033) by [@xPaw](https://github.com/xPaw))
+- Fix duplicate chat containers and unread markers when reconnecting ([#2039](https://github.com/thelounge/lounge/pull/2039) by [@xPaw](https://github.com/xPaw))
+- Fix crash when hostname is changed in lockNetwork mode ([#2042](https://github.com/thelounge/lounge/pull/2042) by [@xPaw](https://github.com/xPaw))
+- Still render link previews if image fails to load ([#2043](https://github.com/thelounge/lounge/pull/2043) by [@xPaw](https://github.com/xPaw))
+- Make sure packages can be referenced in subfolders ([#2045](https://github.com/thelounge/lounge/pull/2045) by [@xPaw](https://github.com/xPaw))
+- Ensure packages loaded are directories ([#2035](https://github.com/thelounge/lounge/pull/2035) by [@astorije](https://github.com/astorije))
+- Fix themes not loading ([#2060](https://github.com/thelounge/lounge/pull/2060) by [@xPaw](https://github.com/xPaw))
```
Comment (xPaw) [inline CHANGELOG.md]: Remove this line, this was only broken by the change one line up.

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
...
+- Fix border after nickname not taking full height ([#2055](https://github.com/thelounge/lounge/pull/2055) by [@xPaw](https://github.com/xPaw))
+- Provide exact version into `npm install` command ([#2063](https://github.com/thelounge/lounge/pull/2063) by [@xPaw](https://github.com/xPaw))
+- Track channel state to allow removing channels user is not in ([#2058](https://github.com/thelounge/lounge/pull/2058) by [@xPaw](https://github.com/xPaw))
+- Allow scaling the page ([#1910](https://github.com/thelounge/lounge/pull/1910) by [@xPaw](https://github.com/xPaw))
+- Fix bind not being passed to `irc-framework` ([#2071](https://github.com/thelounge/lounge/pull/2071) by [@xPaw](https://github.com/xPaw))
```
Comment (xPaw) [inline CHANGELOG.md]: add ` around bind

**Comment by astorije** [comment]
@xPaw, all addressed, thanks!

**Comment by astorije** [comment]
Thanks @MaxLeiter, @McInkay, @milindl, @RockyTV, and @xPaw!

**Pull request merged**